src/index.js

```diff
@@ -1,13 +1,40 @@
-import _ from "lodash";
-import "./styles.css";
+import './styles.css';
 
-function component() {
-  const element = document.createElement("div");
+let taskList = [];
 
-  // Lodash, now imported by this script
-  element.innerHTML = _.join(["Hello", "webpack"], " ");
+function taskCreator(str) {
+  const task = {
+    description: `${str}`,
+    completed: false,
+    index: taskList.length,
+  };
+  taskList = taskList.concat(task);
+}
+
+taskCreator('Task N°1');
+taskCreator('Task N°2');
+taskCreator('Task N°3');
+taskCreator('Task N°4');
+
```

Comment:
- [ OPTIONAL ] You could even create a for loop and call the taskCreator function to render the task[i], the way you are doing manually is not really clean, in my opinion.